api-report/tree2.api.md

```diff
@@ -945,6 +964,19 @@ export interface InsertBindingContext extends BindingContext {
     readonly type: typeof BindingType.Insert;
 }
 
+declare namespace InternalEditableTreeTypes {
```

Comment:
Should this be tagged as `alpha`?

Reply:
API extractor generated this namespace, and choose not to tag it, so I guess not. I think release tags generally only happen at one level, and the things inside this have tags.